portal/src/graphql/portal/BotProtectionConfigurationScreen.tsx

```diff
@@ -895,6 +898,39 @@ const BotProtectionConfigurationContentRequirementsSectionFlowHeader: React.VFC<
     );
   };
 
+interface UseDisabledDetailListOnRenderRowParams {
+  disabled: boolean;
+}
+function useDisabledDetailListOnRenderRow(
+  params: UseDisabledDetailListOnRenderRowParams
+): IRenderFunction<IDetailsRowProps> {
+  const { disabled } = params;
```

Comment:
Read `disabled` from `props` in the render function instead of passing it to this hook